### PR TITLE
Steven_Kuzhipala_Pneumonia_Model_11/13

### PR DESCRIPTION
**Pull request by bobbybendola**
PNEUMONIA MODEL after fixing RGB issues and using sci-kit learn for accuracy plots

part of #5 

I have continued building on the same Pre-Processing notebook from last time. I was able to quickly solve the problem of the black and white images by using TensorFlow, and forcing the images to be loaded as RGB. I have linked my sources, and the suggested data augmentation method from TensorFlow in the notebook.

A good amount of the documentation for the testing and evaluation phase was again from TensorFlow and scikit-learn as well.

From my training accuracy and validation plots, unfortunately, the model seems to be overfitted, as the training accuracy seems to be quite high, and the loss seems to keep going low, meaning that our model seems to have well-learned the training dataset and does not work as well for the new data it sees.

The results from the confusion matrix, the model did identify 31 false negatives, or people who had pneumonia but had been classified as normal, which is not good for a healthcare model.

### CODE REVIEW

**Review by margoglvz — APPROVED** [review]
Hi @bobbybendola! Thank you for submitting your PR and I like how you noticed how false negatives are important for a healthcare model! I think the code and structure looks really good and I think the best next step is to continue testing with different hyperparamteres  (hidden layers, # of epochs, etcs.) and to use that graph you use in your code to see for example, which epoch number would be best for your model. Definitely good you started small but if possible I think you could possibly try up to 50 or even 100! to see which one will work best for your model. Let us know if you have any other questions!
